jenkins/job-configs/kubernetes-jenkins/kubernetes-soak.yaml

```diff
@@ -302,7 +306,6 @@
                 export PROJECT="k8s-jkns-gke-soak"
                 # Need at least n1-standard-2 nodes to run kubelet_perf tests
                 export MACHINE_TYPE="n1-standard-2"
-                export E2E_OPT="--check_version_skew=false"
```

Comment:
So was this just a bug? It seems like we'll always want check skew as false when running tests but not when deploying the cluster? Or do we only want this set to false for gke jobs because we get the version from prod?